lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -15,18 +15,17 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 package OpenQA::Schema::Result::Assets;
-use base qw/DBIx::Class::Core/;
+use base qw(DBIx::Class::Core);
```

Comment:
in some places:

coolo@cleopatra#os-autoinst>man perlfunc | grep 'qw(' | wc -l
19
coolo@cleopatra#os-autoinst>man perlfunc | grep 'qw/' | wc -l
2